data/mock_data.rb

```diff
@@ -0,0 +1,75 @@
+require 'Date'
+
+class MockData
+
+  def self.get_a_random_date(random = true)
+    if random
+      date_s = "20#{rand(10..21)}-#{rand(1..12)}-#{rand(1..28)}"
+      return Date.strptime(date_s, '%Y-%m-%d')
+    else
+      return Date.strptime('2020-01-01', '%Y-%m-%d')
+    end
+  end
+
+  def self.get_a_random_price
+    (rand(1..120) + (rand(100) / 100.0))
+  end
+
+  def self.get_mock_merchants(number_of_mocks: 10, random_dates: true)
+    mocked_merchants = []
+    number_of_mocks.times do |merchant_number|
+      merchant = {}
+      date = get_a_random_date(random_dates)
+
+      merchant[:name] = "Merchant #{merchant_number}"
+      merchant[:id] = merchant_number
+      if block_given?
+        merchant[:created_at] = yield(date).to_s
+        merchant[:updated_at] = date.to_s
+      else
+        merchant[:created_at] = date.prev_year.to_s
+        merchant[:updated_at] = date.to_s
+      end
+      mocked_merchants << merchant
+    end
+    mocked_merchants
+  end
+
+  def self.get_mock_items(number_of_mocks: 10, number_of_merchants: 2, random_dates: true, price_of: 0)
+    mocked_items = []
+    number_of_mocks.times do |item_number|
+      item = {}
+      date = get_a_random_date(random_dates)
+      item[:name] = "Item #{item_number}"
+      item[:id] = item_number
+      if price_of == 0
+        item[:unit_price] = get_a_random_price
+      else
+        item[:unit_price] = price_of
+      end
+      item[:description] = 'Item Description'
+      item[:merchant_id] = item_number % number_of_merchants
+      if block_given?
+        item[:created_at] = yield(date).to_s
+        item[:updated_at] = date.to_s
+      else
+        item[:created_at] = date.prev_year.to_s
+        item[:updated_at] = date.to_s
+      end
+      item[:updated_at] = date.to_s
+      mocked_items << item
+    end
+    mocked_items
+  end
+
+  def self.sum_prices(items)
+    items.sum do |item|
+      item[:unit_price]
+    end
+  end
+
+  def self.mean_price(items)
```

Comment:
Should we do a std_deviation one here too? Or is that excessive? Definitely not needed for this PR, but I suppose if we're going to do that over and over in the BI levels it could be interesting? Haven't seen a use case yet from the stuff I've been focused on.